config/routes.rb

```diff
@@ -375,15 +375,23 @@
           # sometimes underscores get messed up when linked to via SMS
           as: :capture_doc_dashes
 
-      get '/in_person_proofing/ssn' => 'in_person/ssn#show'
-      put '/in_person_proofing/ssn' => 'in_person/ssn#update'
+      # DEPRECATION NOTICE
```

Comment:
@NavaTim Can these be removed now, since these changes have been deployed in production?

Reply:
Yes, the deprecated routes can be removed.

Reply:
@NavaTim Can you take care of cleaning them up? Or is this captured in a ticket somewhere?

Reply:
Work on the FSM is being deprioritized for the moment, but I can make sure this is captured in a ticket.